Source/Diagnostics/ReducedDiags/FieldProbe.H

```diff
@@ -76,6 +76,9 @@ private:
     //! counts number of particles for all MPI ranks
     long m_valid_particles {0};
 
+    //! remember the last time `ComputeDiags` was called to count the number of steps  in between(for non-integrated detectors)
```

Comment:
I think cross-reference might even work :)
```suggestion
    //! remember the last time @see ComputeDiags was called to count the number of steps in between (for non-integrated detectors)
```